tests/functional/spec/features/routing/any_in.spec.js

```diff
@@ -0,0 +1,31 @@
+import CountryCheckboxPage from "../../../generated_pages/new_routing_checkbox_contains_any/country-checkbox.page";
+import CountryInterstitialPage from "../../../generated_pages/new_routing_checkbox_contains_any/country-interstitial-india-or-malta-or-both.page";
+import CountryInterstitialOtherPage from "../../../generated_pages/new_routing_checkbox_contains_any/country-interstitial-not-india-or-malta-or-both.page";
+
+describe("Feature: Routing - ANY-IN Operator", () => {
+  describe("Equals", () => {
+    describe("Given I start the ANY-IN operator routing survey", () => {
+      beforeEach(() => {
+        browser.openQuestionnaire("test_new_routing_checkbox_contains_any.json");
+      });
+
+      it("When I do select India and Malta, Then I should be routed to the interstitial page", () => {
```

Comment:
correct answer interstitial page?

Reply:
Done, added to both relevant tests. Thanks!